tests/test_other.py

```diff
@@ -5894,6 +5894,11 @@ def test_define_modularize(self):
   def test_emconfigure_js_o(self):
     # issue 2994
     for i in [0, 1, 2]:
+      if WINDOWS and i == 0:
+        # EMCONFIGURE_JS=0 means to compile to native, and when doing so it
+        # uses windows struct layout, which differs from linux/mac/wasm, which
+        # hits assertions in the wasi headers.
+        continue
```

Comment:
Hmm,  if we don't support EMCONFIGURE_JS=0 should we not error out in that config?  Or is this just a temporary measure?   Should we add ifdef around the assertions?

Reply:
I think we can just remove that mode entirely, actually. Working on a PR.